Bottom: 245186e9239ad11e768d6b9e022d601f17036196
Top:    deeaab77160569b4a66986b3872777fd33d88ce0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 23:34:08 +0100

Refresh of hid-ps-eps-c-don-t-use-core-dr

---

diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index 5775f60..97d4163 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -175,7 +175,7 @@ set_layer (const char *name, int group)
 }
 
 void
-eps_expose (const BoxType *bounds)
+eps_expose (BoxType *bounds)
 {
   HID *old_gui = gui;
   int group;
@@ -229,11 +229,6 @@ eps_hid_export_to_file (FILE * the_file, HID_Attr_Val * options)
   int i;
   static int saved_layer_stack[MAX_LAYER];
   BoxType region;
-  FlagType save_thindraw;
-
-  save_thindraw = PCB->Flags;
-  CLEAR_FLAG(THINDRAWFLAG, PCB);
-  CLEAR_FLAG(THINDRAWPOLYFLAG, PCB);
 
   f = the_file;
